flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java

```diff
@@ -94,20 +85,9 @@ public void setParentFlowChainId(String parentFlowChainId) {
         this.parentFlowChainId = parentFlowChainId;
     }
 
-    public String getChain() {
-        return chain;
-    }
-
+    @SuppressWarnings("unchecked")
     public Queue<Selectable> getChainAsQueue() {
```

Comment:
Is the annotation still necessary? :smile:

Reply:
yes